PiicoDev_Unified.py

```diff
@@ -6,6 +6,14 @@
 compat_ind = 1
 i2c_err_str = "PiicoDev could not communicate with module at address 0x{:02X}, check wiring"
 
+piicodev_addr = { #A list of all PiicoDev sensors and possible addresses
```

Comment:
I like it, although imho for objects I suggest writing out what each element consists of in the comment instead of stating what the object is for, as that should be given by the naming convention. I suggest:

```
piicodev_addrs = { # "Name of PiicoDev Module": [Matching Address], ...
```